### PR TITLE
Fix Raven Beak damage patching in CONSISTENT_HIGH mode

### PR DESCRIPTION
**Pull request by ArcanoxDragon**
Alter RB damage values in "CONSISTENT_HIGH" mode so wave and plasma actually deal the same damage

### CODE REVIEW

File: open_dread_rando/game_patches.py [hunk 1]
```diff
@@ -34,26 +34,42 @@ def _modify_raven_beak_damage_table(editor: PatcherEditor, mode: str):
...
+        # Wave Beam typically deals 1.6x the damage as Plasma Beam, so its relative factor must be 1/1.6 that of
+        #   Plasma Beam (or 5/8, or 0.625)
+        # Ice Missiles typically deal 1.33x the damage as Super Missiles, so their relative factor must be 1/1.33
+        #   that of Super Missiles (or 3/4, or 0.75)
+
+        # Base factors will yield the correct ratio such that wave and plasma deal identical damage
+        base_factor.fWaveBeamFactor = 0.625
+        base_factor.fChargeWaveBeamFactor = 0.625
+        base_factor.fMeleeChargeWaveBeamFactor = 0.625
+        base_factor.fIceMissileFactor = 0.75
```
Comment (duncathan) [inline open_dread_rando/game_patches.py:56]: can you instead set the counter factors for ice missile to 0.75, and leave the base factor at 1.0? that way it'll retain its bonus in phase 2, but still be equally strong as supers for the counters

**Review by duncathan — CHANGES_REQUESTED** [review]

File: open_dread_rando/game_patches.py (hunk 1 above)
Comment (ArcanoxDragon) [reply open_dread_rando/game_patches.py:56]: Yeah, I've gotta leave for dinner soon, but I will make that change once I get a chance

**Comment by henriquegemignani** [comment]
Did the investigations for this reach an end? Are we missing anything for this PR?

**Review by duncathan — APPROVED** [review]

**Comment by duncathan** [comment]
there's more to investigate for curiosity's sake, but the pr is good to go

**Pull request merged**